generators/iac-ci/index.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+const Generator = require('yeoman-generator');
+const kebabCase = require('lodash/kebabCase');
+
+module.exports = class extends Generator {
+  prompting() {
+    const prompts = [
+      {
+        type: 'input',
+        name: 'pipelineName',
+        message: 'Pipeline name:',
+      },
+      {
+        type: 'input',
+        name: 'awsRegion',
+        message: 'AWS Region:',
+      },
+      {
+        type: 'input',
+        name: 'awsAccountNumber',
+        message: 'AWS Account Number:',
+      },
+      {
+        type: 'input',
+        name: 'gitRepo',
+        message: 'Git respository for build source (HTTPS):',
+      },
+    ];
+
+    return this.prompt(prompts).then(props => {
+      this.props = props;
+    });
+  }
+
+  writing() {
+    const environments = ['dev', 'test'];
+
+    const mappings = {
+      pipelineName: this.props.pipelineName,
+      awsRegion: this.props.awsRegion,
+      awsAccountNumber: this.props.awsAccountNumber,
+      gitRepo: this.props.gitRepo,
+    };
```

Comment:
```suggestion
    const { pipelineName, awsRegion, awsAccountNumber, gitRepo } = this.props
    const mappings = { pipelineName, awsRegion, awsAccountNumber, gitRepo }
```

This might look cleaner. Happy for us still to proceed with your current implementation though

Reply:
That suggestion feature didn't work as well as I thought it would